src/__tests__/index.test.ts

```diff
@@ -36,18 +66,47 @@ describe("E2E Test", () => {
 
       // Instead of being smart and actually trying to calculate the actual values (we would need to subtact the index.test.ts tests), we will just hardcode it, so these needs to be updated if the tests are updated
       // Don't get the values from ./run test (would defeat the purpose of this test), run the tests using npm run test and get the values from there and get coverage from npm run test:coverage
-      expect(totalTests).toBe(48);
-      expect(totalPassed).toBe(48);
-      expect(lineCoverage).toBe(94.99);
+      expect(totalTests).toBe(49);
+      expect(totalPassed).toBe(49);
+      expect(lineCoverage).toBe(95.33);
     }
   });
 
-  it("should process a URL file with a default command", async () => {
-    const { stdout, stderr } = await execAsync("./run myFile.txt");
+  it("should calculate a netscore", async () => {
+    await fs.mkdir(testDir, { recursive: true });
 
-    expect(stderr).toBe("");
-    expect(stdout).toContain("Command TBD");
-  });
+    const consoleSpy = vi.spyOn(console, "log").mockImplementation(() => {});
+
+    //////REPLACE WITH ACTUAL EXPECTED VALUES////////
+    const expected = {
+      URL: "https://github.com/cloudinary/cloudinary_npm",
+      NetScore: 0.19,
+      NetScore_Latency: -1,
+      RampUp: 0.27,
+      RampUp_Latency: -1,
+      Correctness: 0,
+      Correctness_Latency: -1,
+      BusFactor: -1,
+      BusFactor_Latency: -1,
+      ResponsiveMaintainer: 0.09,
+      ResponsiveMaintainer_Latency: -1,
+      License: 1,
+      License_Latency: -1
+    };
+
+    const filePath = path.join(testDir, "sampleURL.txt");
+
+    await fs.writeFile(filePath, `https://github.com/cloudinary/cloudinary_npm`);
+
+    await calculateNetScore(filePath);
+
+    expect(calculateCorrectness).toHaveBeenCalledWith("cloudinary", "cloudinary_npm");
+    expect(calculateRampUpScore).toHaveBeenCalledWith("cloudinary", "cloudinary_npm");
+    expect(calculateResponsiveMaintainerScore).toHaveBeenCalledWith("cloudinary", "cloudinary_npm");
+    expect(calculateLicenseScore).toHaveBeenCalledWith("cloudinary", "cloudinary_npm");
+
+    expect(consoleSpy).toHaveBeenCalledWith(expected);
+  }, 50000);
```

Comment:
have this actually call the bash script rather than calling the netscore function